firebase-sessions/src/test/kotlin/com/google/firebase/sessions/ProcessDataManagerTest.kt

```diff
@@ -100,6 +100,53 @@ internal class ProcessDataManagerTest {
     assertThat(coldStart).isFalse()
   }
 
+  @Test
+  fun isMyProcessStale() {
+    val appContext =
+      FakeFirebaseApp(processes = listOf(myProcessInfo)).firebaseApp.applicationContext
+    val processDataManager = ProcessDataManagerImpl(appContext, FakeUuidGenerator(UUID_1))
+
+    val myProcessStale =
+      processDataManager.isMyProcessStale(mapOf(MY_PROCESS_NAME to ProcessData(MY_PID, UUID_1)))
+
+    assertThat(myProcessStale).isFalse()
+  }
+
+  @Test
+  fun isMyProcessStale_otherProcessCurrent() {
+    val appContext =
+      FakeFirebaseApp(processes = listOf(myProcessInfo, otherProcessInfo))
```

Comment:
IIUC the first element in the processes list is assumed as `myProcess` in the rest of the test. Would it be possible to add a comment in `FakeFirebaseApp` about the details?

Reply:
The order of the list does not matter. It looks up processes by the process name. I could change this to a set to make it clearer if you think that matters. Everything I named "my" is self consistent, for example, myProcessName and myPid match myProcessInfo.